sys/arm64/arm64/trap.c

```diff
@@ -273,7 +280,18 @@ cap_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
 		print_registers(frame);
 		printf(" far: %16lx\n", far);
 		printf(" esr:         %.8lx\n", esr);
-		panic("Capability abort from kernel space!");
+		panic("Capability abort from kernel space: %s",
+		    cheri_fsc_string(esr & ISS_DATA_DFSC_MASK));
+	}
+
+	if (log_user_cheri_exceptions) {
+		printf("pid %d tid %d (%s) uid %d: capability abort, %s\n",
+		    td->td_proc->p_pid, td->td_tid, td->td_proc->p_comm,
+		    td->td_ucred->cr_uid,
+		    cheri_fsc_string(esr & ISS_DATA_DFSC_MASK));
+		print_registers(frame);
+		printf(" far: %16lx\n", far);
+		printf(" esr:         %.8lx\n", esr);
```

Comment:
Why the differing indentation?

Reply:
Convention taken from elsewhere so everything is right-aligned

Reply:
```suggestion
		printf(" far: %16lx\n", far);
		printf(" esr: % 16.8lx\n", esr);
```
Maybe make that part of the format?

Reply:
Perhaps better, but then inconsistent with the other 9 instances of this in the file.

Reply:
Maybe should tidy these in a later commit and upstream the others.